server/controllers/index.js

```diff
@@ -41,12 +41,12 @@ export const articles = async(ctx) => {
   });
 };
 
-function postsToPromos(posts) {
+function postsToPromos(posts, weight) {
   return posts.map(articlePromo => {
     const promo: Promo = {
       modifiers: [],
       article: articlePromo,
-      meta: {}
+      weight: weight
```

Comment:
Nice, I can definitely see `BodyPart` and `Promo` basically being the same thing.